src/Jackett/JackettModule.cs

```diff
@@ -59,6 +59,10 @@ protected override void Load(ContainerBuilder builder)
             Mapper.CreateMap<WebClientByteResult, WebClientStringResult>().ForMember(x => x.Content, opt => opt.Ignore()).AfterMap((be, str) =>
             {
                 str.Content = Encoding.UTF8.GetString(be.Content);
+                if(str.Content.Contains("NorBits"))

```

Comment:
Yeah. Me neither. To populate the encoding from the indexer side does sound like a good solution. Most of the site is UTF8 so the old will work.